data/hai/hai reveal 2 start.txt

```diff
@@ -223,6 +292,29 @@ mission "Hai Rescue: Ooonem"
 		fleet "Small Core Pirates" 1
 
 
+mission "Hai Rescue: Meroot and Lucy Hint"
+	landing
+	name "Deep Crevasses"
+	description "There could be kidnappers or kidnapped Hai hiding on Canyon."
+	source
+		near "Zaurak" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+	destination Canyon
+	to offer
+		"combat rating" > 50
+		has "Unwanted Cargo: done"
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Meroot and Lucy: offered"
+		not "event: hai-human treaty signed"
+		random < 80
+	on offer
+		conversation
+			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is 'deep crevasses' on <destination> and it occurs to you that it would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
```

Comment:
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is "deep crevasses" on <planet>, it occurs to you that there would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
```
`Canyon` rather than `Canyon in the soandso system` (can't remember the name of it rn) seems to fit better to me.